app/Jobs/ElasticSearchIndexInit.php

```diff
@@ -63,15 +84,40 @@ public function handle()
                 new \RuntimeException('wbstackElasticSearchInit call for '.$this->wikiDomain.'. No wbstackElasticSearchInit key in response: '.$rawResponse)
             );
 
-            return; //safegaurd
+            return;
         }
 
         if ($response['wbstackElasticSearchInit']['success'] == 0) {
             $this->fail(
                 new \RuntimeException('wbstackElasticSearchInit call for '.$this->wikiDomain.' was not successful:'.$rawResponse)
             );
 
-            return; //safegaurd
+            return;
+        }
+
+        $output = $response['wbstackElasticSearchInit']['output'];
+
+        $newlyCreated = "\tCreating index...ok"; // occurs a couple of times when newly created
+        $updated = "\t\tValidating {$this->wikiDomain}_general alias...ok"; // occurs on a successful update run
+
+        $enableElasticSearchFeature = false;
+
+        if ( in_array( $newlyCreated,  $output ) ) {
+
+            // newly created index succeeded, turn on the wiki setting
+            $enableElasticSearchFeature = true;
+
+        } else if ( in_array( $updated, $output ) ) {
+
+            // script ran and update was successful, make sure feature is enabled
+            $enableElasticSearchFeature = true;
         }
```

Comment:
We probably want to log some kind of warning if neither of these conditions is true?

Reply:
done